test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
```

Comment:
Can you use the [standard header](https://github.com/nodejs/node/blob/master/test/parallel/test-whatwg-url-searchparams-set.js#L1-L12) for WPT tests?